R/strings.R

```diff
@@ -53,3 +53,19 @@ pkg.env$node_attr.dataType.float <- "float"
 pkg.env$node_attr.property.valid <- "valid"
 pkg.env$node_attr.property.invalid <- "invalid"
 pkg.env$node_attr.property.missing <- "missing"
+
+pkg.env$columns.Variables <- "variable"
```

Comment:
Yeah variables was just the relic name and since we introduced this sheet i only changed values not names but this is a good time to change it. Thanks

Reply:
I dont think units works since its for the units column which can be more then 1 type of unit